src/byrow.jl

```diff
@@ -51,31 +51,27 @@ function byrow_helper(df, body)
     (e_body, e_newcols) = byrow_find_newcols(body, Any[])
     e_delimiters = Expr(:(=), :row, :( 1:_N ))
     e_forloop = Expr(:for, e_delimiters, byrow_replace(e_body))
-    return quote
-        _N = length($df[1])
-        _DF = @transform($df, $(e_newcols...))
-        @with _DF $e_forloop
-        _DF
-    end
+    :(_N = length($df[1]); _DF = @transform($df, $(e_newcols...)); @with _DF $e_forloop; _DF)
```

Comment:
Well, user code after a macroexpand would contain line numbers from the byrow.jl file. Actually, I guess that's probably a good thing for backtraces?, but maybe due to a bug, these lines would show up as uncovered. Right now coverage for this pull request is up to like 97% in part because of changes I made like this.